runtime/jvmti/jvmtiStartup.c

```diff
@@ -374,7 +374,36 @@ issueAgentOnLoadAttach(J9JavaVM * vm, J9JVMTIAgentLibrary * agentLibrary, const
 		invocationJavaVM = (J9InvocationJavaVM *)vm;
 	}
 
-	rc = agentInitFunction(invocationJavaVM, (char *) options, NULL);
+#ifdef WIN32
+	{
+		/* agentInitFunction invokes java.instrument/share/native/libinstrument/InvocationAdapter.c
+		 * which expects jarPath in system default code page encoding.
+		 */
+		char *jarPath = NULL;
+		BOOLEAN conversionSucceed = FALSE;
+		int32_t size = j9str_convert(J9STR_CODE_MUTF8, J9STR_CODE_WINDEFAULTACP, options, strlen(options), NULL, 0);
```

Comment:
If `rc = JNI_ERR;` is set here, then the `else` clause below is unnecessary

Reply:
Right, initialized `rc = JNI_ERR` and avoid setting it in `else` clause.